main.nf

```diff
@@ -0,0 +1,60 @@
+nextflow.enable.dsl = 2
+
+include { EXPORT_RAW_GENOMES } from "./modules/export_raw_genomes/export_raw_genomes.nf"
+include { FASTQC as FASTQC_ORIGINAL } from "./modules/fastqc/fastqc.nf" addParams(resultsDir: "${params.outdir}/fastqc/original")
+include { FASTQC as FASTQC_TRIMMED } from "./modules/fastqc/fastqc.nf" addParams(resultsDir: "${params.outdir}/fastqc/trimmed")
```

Comment:
We discussed this on 29th June, the same file can be used for creating both aliases.

Reply:
yes! And It makes more sense than using different files.